setup.py

```diff
@@ -41,6 +41,7 @@ def read_file(filename, encoding='utf8'):
     'ConfigArgParse>=0.9.3',
     'configobj',
     'cryptography>=1.2',  # load_pem_x509_certificate
+    'josepy',
```

Comment:
Looks good, but it seems to me we should also pin <2.0.

Reply:
I thought we were in agreement in #4433 not to pin major versions back like this and instead do so in `venv.sh`?

Reply:
Ah, yes, I forgot about that. r+